local/test-binary

```diff
@@ -19,25 +19,27 @@ rm -fr ${WORKDIR_ABS}/PS
 mkdir -p ${WORKDIR_ABS}/PS
 tar -C ${WORKDIR_ABS}/PS --strip-components=1 -zxpf $(ls $WORKDIR_ABS/*.tar.gz | head -1)
 cd ${WORKDIR_ABS}/PS/mysql-test
-#
-# CentOS 6 and 7
-if [[ -f /opt/rh/devtoolset-7/enable ]]; then
-  source /opt/rh/devtoolset-7/enable
-fi
-#
 TESTCASE_TIMEOUT=60
 PARALLEL=$(grep -c ^processor /proc/cpuinfo)
+
 # CentOS 6 & 7
 if [[ -f /opt/rh/devtoolset-7/enable ]]; then
-        source /opt/rh/devtoolset-7/enable
+    source /opt/rh/devtoolset-7/enable
 fi
-#
+
+# Centos 6/i386
+if [[ $(rpm --eval %_arch) = "i386" ]] && [[ -f /opt/rh/devtoolset-2/enable ]]; then
```

Comment:
Shouldn't this and above be merged to
if 386 && -f devtoolset-2 -f
   enable -2
elif -f devtoolset-7
   enable -7
fi
?

Reply:
it's not from my commits. @hors ?

Reply:
fixed, restarting tests

Reply:
Does not look fixed?

Reply:
@laurynas-biveinis devtoolset logic is not related to JEN-960 and this PR was created before any work on devtoolset-3 logic was initiated.
1 issue - 1 PR, isn't it?

Reply:
Perhaps the PR needs rebasing then? I see that this PR adds devtoolset-2, if it does, it has to to fix the devtoolset choice logic, because it adds that choice